samples/server/go-server/api/server/storage/pgsqlstore.go

```diff
@@ -0,0 +1,492 @@
+// Copyright 2017 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/grafeas/grafeas/samples/server/go-server/api/server/name"
+	pb "github.com/grafeas/grafeas/v1alpha1/proto"
+	"github.com/lib/pq"
+	opspb "google.golang.org/genproto/googleapis/longrunning"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type PgSQLConfig struct {
```

Comment:
What are your thoughts on putting the config and creation into a separate file? 

These are things that aren't going to be modified often in the normal course of development, however the create, insert, delete, list methods will be.

Reply:
Fixed in c551fed